### PR TITLE
Add recipe for conduction/commonground-bundle

### PR DESCRIPTION
**Pull request by rubenvdlinde**
| Q             | A
| ------------- | ---
| License       | MIT

We are setting up a new bundle for use with dutch governemental api's, the bundle is in progres but the first service (kadaster) now has a release candidate so we would like to move foreward with placing this bundle on symfony flex.

We tried to tackle as much of the contributing guidelines as we could, but are always open for feedback!

### CODE REVIEW

File: conduction/commonground-bundle/config/conduction_common_ground.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        apikey: 'ChangeMe' 
+        # The default KVK api location, change this if yoy are using an NLX adaptor, but keep in mind that the kvk uses port 443
+        location: 'https://api.kvk.nl:443/' 
+        
+services:    
+    # please note that last definitions always *replace* previous ones
+    conduction.commonground.bag:
+        class: App\Conduction\CommonGroundBundel\Service\KadasterService
+        arguments: 
+            - []
```
[the remark this answers is not in the document]
Comment (rubenvdlinde) [reply conduction/commonground-bundle/config/conduction_common_ground.yaml]: Added a new line

File: conduction/commonground-bundle/manifest.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    "copy-from-recipe": {
+        "Resources/config/": "%CONFIG_DIR%/packages",
+        "Resources/public/": "%PUBLIC_DIR%/bundles/commonground",
+    },
+    "env": {
+        "CONDUCTION_COMMONGROUND_BAG_APIKEY": "",
+        "CONDUCTION_COMMONGROUND_KVK_APIKEY": ""
+    },
+    "aliases": ["commonground"]
+}
```
[the remark this answers is not in the document]
Comment (rubenvdlinde) [reply conduction/commonground-bundle/manifest.json]: Added a new line

File: conduction/commonground-bundle/post-instal.txt [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+<bg=blue;fg=white> What's next? </>
+<bg=blue;fg=white>              </>
+
+  * Your first component is almost ready!
+    1. Create your first API resource in <info>src/Entity</info>;
+    2. Go to <info>/api</info> to browse your API
+
+  * <fg=blue>Read</> the platform documentation at <comment>https://common-ground.dev/docs</>
+
+  * <fg=blue>Read</> the commonground documentation at <comment>https://commonground.nl</>
```
[the remark this answers is not in the document]
Comment (rubenvdlinde) [reply conduction/commonground-bundle/post-instal.txt]: Added a new line

File: conduction/commonground-bundle/post-instal.txt [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
```
[the remark this answers is not in the document]
Comment (rubenvdlinde) [reply conduction/commonground-bundle/post-instal.txt]: Added a new line

**Comment by rubenvdlinde** [comment]
Its a bit unclear to me what to make of the  "Recipe must define a "manifest.json" file" validation error, since a manifest.json file is present.

File: conduction/commonground-bundle/config/conduction_common_ground.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
+conduction_common_ground:      
+    bag:
+        # cashing is a common ground hot patato, from a data at the source perspective we don't want to duplicate data
```
Comment (nicolas-grekas) [inline conduction/commonground-bundle/config/conduction_common_ground.yaml]: I don't know anything about the bundle, yet I'm not sure this comment helps me in any way, sorry :)
Any more factual alternatives (or is the bundle about patatoes? )

File: conduction/commonground-bundle/config/conduction_common_ground.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+        # but.... city and street names are not going to change regularly, so to prevent unnecessary api calls cashing makes sense
+        #
+        # Therefore we are going to allow cashing on the bag api
+        #
+        # Valid values:
+        # - false (no cashing)
+        # - integer (cash for given amount of milliseconds)
+        # - a valid php datetime input (cash item deleted at that date), https://www.php.net/manual/en/function.strtotime.php
+        cash:
```
Comment (nicolas-grekas) [inline conduction/commonground-bundle/config/conduction_common_ground.yaml]: why is this not already the default in the bundle?

File: conduction/commonground-bundle/config/conduction_common_ground.yaml [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+        cash:        
+            nummeraanduiding: 604800000 # 1 week      
+            woonplaats: 'January 1st Next Year' # By law Dutch localities only change in January the first      
+            openbareruimte:  604800000 # 1 week      
+            panden: 604800000 # 1 week
+            verblijfsobject: 604800000 # 1 week
+            ligplaatsen: 604800000 # 1 week
+            standplaatsen: 604800000 # 1 week
+        # Get your BAG api key at https://bag.basisregistraties.overheid.nl/restful-api, or by sending a email to dataplatform@kadaster.nl        
+        apikey: 'ChangeMe'
```
Comment (nicolas-grekas) [inline conduction/commonground-bundle/config/conduction_common_ground.yaml]: should it be an env var? see how other recipes  use `%env(FOO)%` references + declare them in manifest.json

File: conduction/commonground-bundle/config/conduction_common_ground.yaml [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+            woonplaats: 'January 1st Next Year' # By law Dutch localities only change in January the first      
+            openbareruimte:  604800000 # 1 week      
+            panden: 604800000 # 1 week
+            verblijfsobject: 604800000 # 1 week
+            ligplaatsen: 604800000 # 1 week
+            standplaatsen: 604800000 # 1 week
+        # Get your BAG api key at https://bag.basisregistraties.overheid.nl/restful-api, or by sending a email to dataplatform@kadaster.nl        
+        apikey: 'ChangeMe' 
+        # The default BAG api location, change this if yoy are using an NLX adaptor
+        location: 'https://bag.basisregistraties.overheid.nl/api/v1/'
```
Comment (nicolas-grekas) [inline conduction/commonground-bundle/config/conduction_common_ground.yaml]: if this is the default, this should be removed - if it's not the default, should it be it?

File: conduction/commonground-bundle/config/conduction_common_ground.yaml [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+            standplaatsen: 604800000 # 1 week
+        # Get your BAG api key at https://bag.basisregistraties.overheid.nl/restful-api, or by sending a email to dataplatform@kadaster.nl        
+        apikey: 'ChangeMe' 
+        # The default BAG api location, change this if yoy are using an NLX adaptor
+        location: 'https://bag.basisregistraties.overheid.nl/api/v1/' 
+    kvk:
+        # Get your KVK api key at https://www.kvk.nl/producten-bestellen/koppeling-handelsregister/kvk-api/        
+        apikey: 'ChangeMe' 
+        # The default KVK api location, change this if yoy are using an NLX adaptor, but keep in mind that the kvk uses port 443
+        location: 'https://api.kvk.nl:443/'
```
Comment (nicolas-grekas) [inline conduction/commonground-bundle/config/conduction_common_ground.yaml]: same comments as above

File: conduction/commonground-bundle/config/conduction_common_ground.yaml [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+        # The default BAG api location, change this if yoy are using an NLX adaptor
+        location: 'https://bag.basisregistraties.overheid.nl/api/v1/' 
+    kvk:
+        # Get your KVK api key at https://www.kvk.nl/producten-bestellen/koppeling-handelsregister/kvk-api/        
+        apikey: 'ChangeMe' 
+        # The default KVK api location, change this if yoy are using an NLX adaptor, but keep in mind that the kvk uses port 443
+        location: 'https://api.kvk.nl:443/' 
+        
+services:    
+    # please note that last definitions always *replace* previous ones
```
Comment (nicolas-grekas) [inline conduction/commonground-bundle/config/conduction_common_ground.yaml]: which means this very declaration *will* be replaces, as it's in the `App` namespace, that is loaded after this file

File: conduction/commonground-bundle/public/img/vng.svg [hunk 11]
```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (nicolas-grekas) [inline conduction/commonground-bundle/public/img/vng.svg]: all images should be in the bundle

**Comment by Nyholm** [comment]
Friendly ping

**Pull request closed without merging**